flexget/plugins/filter/unique.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+
+from flexget import plugin
+from flexget.event import event
+
+log = logging.getLogger('unique')
+
+
+class Unique(object):
+    """
+    Take action on entries with duplicate fields, except for the first item
+
+    Reject the second+ instance of every movie:
+
+      unique:
+        field:
+          - imdb_id
+          - movie_name
+        action: reject
+    """
+
+    schema = {
+        'oneOf': [
+            {'type': 'boolean'},
+            {
+                'type': 'object',
+                'properties': {
+                    'field': {
+                        'oneOf': [
+                            {'type': 'string'},
+                            {'type': 'array'},
+                        ]
+                    },
+                    'action': {'enum': ['accept', 'reject']},
+                },
+                'required': [],
```

Comment:
OK, I've misremembered.
I'm feeling like if you say "I want these entities to be unique" and you're not saying based on which criteria, it's legit to compare the entire entity.
If you feel like this is a weird behavior, I'll remove it and mandate the field